feature/mpls/static/otg_tests/static_bgp_nexthop/README.md

```diff
@@ -0,0 +1,80 @@
+# MPLS-2.2: MPLS forwarding via static LSP to BGP next-hop.
+
+## Summary
+
+Validate static LSP functionality with BGP resolved next-hop.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_4.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed)
+
+## Procedure
+
+### Configuration
+
+1) Create the topology below:
+
+    ```
+                         |         | ---- | ATE Port 2 | ---- [eBGP peer]
+    [ ATE Port 1 ] ----  |   DUT   |      |            |
+                         |         | ---- | ATE Port 3 |
+    ```
+
+2)  Configure eBGP peer on ATE Port 2 interface and advertise BGP-NH-V4 and BGP-NH-V6
+3)  Configure discard static routes for BGP-NH-V4 and BGP-NH-V6 to Null0 via administrative distance of 254.
+4)  Enable MPLS forwarding.
+5)  Create egress static LSP for IPv4 traffic to pop the label and resolve the next-hop BGP-NH
+    *  Match incoming label (1000004)
+    *  Set the action to pop label
+    *  Set IPv4 next-hop and resolve NH action
+6)  Create egress static LSP for IPv6 traffic to pop the label and resolve the next-hop BGP-NH
+    *  Match incoming label (1000006)
+    *  Set the action to pop label
+    *  Set IPv4 next-hop and resolve NH action
+7)  Configure cover static routes to IPV4-DST and IPV6-DST to ATE Port 3.
```

Comment:
Since this portion of the config is the key focus of the test, instead of providing a description of the configuration in English the config should be provided in OC.  I recommend typing it in yaml format because this is maybe easier than JSON.

```yaml
network-instances:
  - network-instance:
    mpls:
      lsps:
        static-lsps:
          - static-lsp:
            config:
              name: "lsp-egress-v4"
            egress:
              next-hop: BGP-NH-V4
              incoming-label: 1000004
          - static-lsp:
            config:
              name: "lsp-egress-v6"
            egress:
              next-hop: BGP-NH-V6
              incoming-label: 1000006
```